src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -126,13 +126,15 @@ export function refreshTransaction(
         });
 
         if (isFailedTransaction(transactionRefreshed) && analyticsSectionName) {
+          const errorMetadata =
+            transactionRefreshed?.data?.attributes?.metadata?.[0] ?? {};
+          const errorCode = errorMetadata.code ?? 'unknown-code';
```

Comment:
Setting this to `unknown-code` should make it easier for us to see when the new data recording shows up in GA.